content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,174 @@ import InfoBox from 'shared/components/InfoBox';
-        "autoscaling:DescribeAutoScalingInstances",
-        "autoscaling:DescribeLaunchConfigurations",
-        "autoscaling:DescribeTags",
-        "ec2:DescribeInstanceTypes",
-        "ec2:DescribeLaunchTemplateVersions"
-      ],
-      "Resource": ["*"]
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:SetDesiredCapacity",
-        "autoscaling:TerminateInstanceInAutoScalingGroup",
-        "ec2:DescribeInstanceTypes",
-        "eks:DescribeNodegroup"
-      ],
-      "Resource": ["*"]
-    }
-  ]
-}
-```
-<InfoBox>
+# AWS Cluster Autoscaler 
 
-To deploy AWS Autoscaler, update the Kubernetes pack `managedMachinePool` node group with the **ARN** of the autoscaler policy created for the service account.
+Palette supports autoscaling for EKS clusters by using the AWS Cluster Autoscaler pack. 
+The AWS Cluster Autoscaler dynamically scales cluster resources. It monitors the workload and provisions or shuts down cluster nodes to maximize cluster utilization. It resizes the Kubernetes cluster under the following conditions:
+<br />
 
-```yaml
-managedMachinePool:
-  #roleName: {{ name of the self-managed role | format "${string}" }}
+* Insufficient resources leading to multiple pod failures in the cluster. The AWS Cluster Autoscaler redistributes these pods to different nodes in such cases.
 
-  ## A list of additional policies to attach to the node group role
-  roleAdditionalPolicies:
-  - "arn:aws:iam::012345678910:policy/autoscalingpolicy"
-```
-</InfoBox>
 
+* Underutilized cluster nodes for a specific period. In this scenario, the AWS Cluster Autoscaler migrates the pods from underutilized nodes to other available nodes.
 
-## Versions Supported
+# Versions Supported
 
 <Tabs>
 
-<Tabs.TabPane tab="1.22.x" key="1.22.x">
+<Tabs.TabPane tab="1.26.x" key="1.26.x">
+
+## Prerequisites
+
+* Kubernetes 1.24.x or higher. 
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="1.24.x" key="1.24.x">
+
+## Prerequisites
 
-**1.22.2**
+* Kubernetes 1.19.x or higher. 
 
 </Tabs.TabPane>
 
 <Tabs.TabPane tab="1.0.x" key="1.0.x">
 
-**1.0.0**
+## Prerequisites
+
+* Kubernetes 1.19.x or higher. 
 
 </Tabs.TabPane>
+
 </Tabs>
 
-## References
+* Permission to create an IAM policy in the AWS account you use with Palette. 
+
+
+* **IAM policy** - The following full Cluster Autoscaler policy should be attached to the cluster's node group. There are two ways to achieve this prerequisite. You can define this as a *customer-managed* policy in the AWS account and use its Amazon Resource Name (ARN) in the cluster profile. Alternatively, you can attach the following IAM policy as an *inline* policy to the node group if you have already deployed your cluster. Refer to the **Usage** section below for more details. <Tooltip>Refer to the [AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html) for the differences between managed and inline policies.</Tooltip>

```

Comment:
```suggestion
*IAM policy - The following IAM policy must be attached to the EKS cluster's node group. There are two ways to achieve this prerequisite. You can define the policy as a *customer-managed* policy in the AWS account and use its Amazon Resource Name (ARN) in the cluster profile. Alternatively, you can attach the following IAM policy as an *inline* policy to the node group if you have already deployed your cluster. Refer to the **Usage** section below for more details. 
```

I removed the bold because it's not needed, and we also don't use tooltips. Link to the usage section.  You can link to an element inside the tabs, refer to [this](https://github.com/spectrocloud/librarium#tabs-component) section of the docs repo's README.